insights/parsers/tests/test_du_usage.py

```diff
@@ -0,0 +1,62 @@
+from insights.parsers.du_usage import DiskUsage
+from insights.tests import context_wrap
+
+du_content = """
+22G     /var/lib/mongodb
+4.0K	/var/lib/pulse
+113M	/var/lib/rpm
+4.0K	/var/lib/samba/private
+8.0K	/var/lib/samba
+8.0K	/var/lib/xkb
+4.0K	/var/lib/gssproxy/clients
+4.0K	/var/lib/gssproxy/rcache
+12K	/var/lib/gssproxy
+4.0K	/var/lib/net-snmp/cert_indexes
+4.0K	/var/lib/net-snmp/mib_indexes
+12K	/var/lib/net-snmp
+56K	/var/lib/upower
+4	/root
+92	/opt/google/chrome/default_apps
+60320	/opt/google/chrome/swiftshader
+18696	/opt/google/chrome/locales
+267032	/opt/google/chrome
+267036	/opt/google
+""".strip()
+
+
+def test_du_disk_usage():
+    context = context_wrap(du_content)
+    du_info = DiskUsage(context)
+
+    assert len(du_content) == 482
```

Comment:
Why are you asserting something about the input data?

Reply:
I was using that to test something, removed now.